web/tailwind.config.js

```diff
@@ -3,6 +3,9 @@ module.exports = {
     darkMode: false, // or 'media' or 'class'
     theme: {
         extend: {
+            borderRadius: {
+                '4xl': '2.5rem',
+            },
```

Comment:
Do we need this?

Reply:
@asasmith  yes, josh requested we change the border radius to 2.5. TW's class only goes up to 1.5

Reply:
@kcbreuil can't you do the same as you did for the `cardStyle` declaration in `/components/card.js`?

Reply:
@asasmith it didnt extend

Reply:
that seems odd. Are you seeing the class applied correctly?

Reply:
`rounded-bl[2.5rem]` seems like it should work in both places.

Reply:
Nope, it doesn't. It applies it in the html but doesnt extend. Creating the custom class in the config does. @asasmith

Reply:
it works
![Screen Shot 2021-06-10 at 12 20 35 PM](https://user-images.githubusercontent.com/11489171/121561199-4576b280-c9e6-11eb-8eac-159a41601609.png)

Reply:
![Screen Shot 2021-06-10 at 12 21 57 PM](https://user-images.githubusercontent.com/11489171/121561418-7656e780-c9e6-11eb-94a4-76ea7a5f0bc2.png)